tests/node/rpc/messages/OnVehicleDataTests.js

```diff
@@ -67,11 +86,13 @@ describe('OnVehicleDataTests', function () {
         const testHandsOffSteering = rpcMessage.getHandsOffSteering();
         const testWindowStatus = rpcMessage.getWindowStatus();
         const testGearStatus = rpcMessage.getGearStatus();
+        const testClimateData = rpcMessage.getClimateData();
 
         // Valid Tests
         Validator.assertEquals(this.stabilityControlsStatus, testStabilityControlsStatus);
         Validator.assertEquals([Test.GENERAL_WINDOW_STATUS], testWindowStatus);
```

Comment:
**Optional**: This is not related to this PR, but this line is missing and can be added on line 93 (above windowStatus)
      `Validator.assertEquals(Test.GENERAL_BOOLEAN, testHandsOffSteering);`.

Reply:
Based on previous experience, it is better to avoid changes not related to PR. Will add if PMs request this in the review.